specs/toolsets/adr_base/setup-toolset.sh

```diff
@@ -58,43 +43,30 @@ case $HAM_OS in
         fi
         ;;
     OSX)
-        export ADR_NDK_PREBUILT=darwin-x86_64
-        export ADR_NDK_VERSION=r12b
-        export GCC_VER=4.9
+        export SYSTEM_NAME=$(uname)-$(uname -m)
+        export ADR_NDK_PREBUILT=${SYSTEM_NAME,,}
         export ADR_DIR_SDK="/usr/local/share/android-sdk"
+        export ADR_DIR_NDK="${ADR_DIR_SDK}/ndk/${ADR_NDK_VERSION}"
         chmod +x "${HAM_TOOLSET_DIR}/adr-"*
 
         # Test the SDK
         if [ ! -d "$ADR_DIR_SDK" -o ! -d "${ADR_DIR_SDK}/platforms/${ADR_SDK_PLATFORM}" ]; then
             echo "I/Can't find android sdk, installing with brew..."
-            ham-brew cask install android-sdk android-platform-tools
-            echo "I/Making sure the Android SDK isn't quarantined..."
-            sudo xattr -r -d com.apple.quarantine /usr/local/Caskroom/android-sdk/
-            sudo xattr -r -d com.apple.quarantine /usr/local/Caskroom/android-platform-tools/
+            brew install --cask android-sdk android-platform-tools
+            # echo "I/Making sure the Android SDK isn't quarantined..."
+            # sudo xattr -r -d com.apple.quarantine /usr/local/Caskroom/android-sdk/
```

Comment:
This was added for a reason, its possible its not needed anymore, but I would keep it since it was added after noticing those failures. Needs to use something like `` DIR=`ham-brew-installdir android-sdk` `` to retrieve the correct directory - it is different on x86 & arm macs.